include/opendht/network_engine.h

```diff
@@ -362,10 +362,14 @@ class NetworkEngine final {
 
     void blacklistNode(const std::shared_ptr<Node>& n);
 
+    std::vector<std::shared_ptr<Node>> getNodesFromCache(const InfoHash& id, sa_family_t sa_f);
```

Comment:
Since this method is only a wrapper for `NodeCache::getCachedNodes`, you may as well call thsi method ^ the same.